GaelAlburo/backend-project
# Prepare Payments API for Containerized Deployment

**Description:**  
This PR focuses on enabling containerized deployment for the Payments API. It introduces a `Dockerfile` and a `requirements.txt` file to facilitate container creation and dependency management. Additionally, import paths within the API have been updated to ensure compatibility when running in a containerized environment.

**Changes:**
- Refactored import paths to ensure they function properly within a containerized environment.  
- Created a `Dockerfile` for building the Payments API container.  
- Added a `requirements.txt` file for managing dependencies.  
- Included a log file for improved debugging and runtime monitoring.  

**Impact:**  
- Enables the Payments API to be deployed as a Docker container.  

**Testing:**  
1. Build the Payments API Docker image and verify successful creation.  
2. Run the containerized Payments API and confirm its functionality.
